src/services/item/plugins/invitation/service.ts

```diff
@@ -54,37 +54,26 @@ export class InvitationService {
   }
 
   async sendInvitationEmail({ member, invitation }: { member: Member; invitation: Invitation }) {
-    const { item, email } = invitation;
+    const { item } = invitation;
 
     // factor out
-    const lang = member.lang;
     const link = buildInvitationLink(invitation);
-
-    const t = this.mailerService.translate(lang);
-
-    const text = t(MAIL.INVITATION_TEXT, {
-      itemName: item.name,
-      creatorName: member.name,
-    });
-    const html = `
-      ${this.mailerService.buildText(text)}
-      ${this.mailerService.buildButton(link, t(MAIL.SIGN_UP_BUTTON_TEXT))}
-      ${this.mailerService.buildText(
-        t(MAIL.USER_AGREEMENTS_MAIL_TEXT, {
-          signUpButtonText: t(MAIL.SIGN_UP_BUTTON_TEXT),
-          graaspLandingPageOrigin: GRAASP_LANDING_PAGE_ORIGIN,
-        }),
-        // Add margin top of -15px to remove 15px margin bottom of the button.
-        { 'text-align': 'center', 'font-size': '10px', 'margin-top': '-15px' },
-      )}
-    `;
-    const title = t(MAIL.INVITATION_TITLE, {
-      itemName: item.name,
-    });
-
-    const footer = this.mailerService.buildFooter(lang);
-
-    this.mailerService.sendEmail(title, email, link, html, footer).catch((err) => {
+    const mail = new MailBuilder({
+      subject: {
+        text: MAIL.INVITATION_TITLE,
+        translationVariables: { itemName: item.name },
+      },
+      lang: member.lang,
+    })
+      .addText(MAIL.INVITATION_TEXT, {
+        itemName: item.name,
+        creatorName: member.name,
+      })
+      .addButton(MAIL.SIGN_UP_BUTTON_TEXT, link)
+      .addUserAgreement(MAIL.SIGN_UP_BUTTON_TEXT)
+      .build();
+
+    this.mailerService.send(mail, member.email).catch((err) => {
```

Comment:
I think this is an error. We are supposed to send the email to the email that was invited not to the user that created the invitation. Does a test ensure this ? It should have broken. 
```suggestion
    this.mailerService.send(mail, invitation.email).catch((err) => {
```

Reply:
Nice catch. I have also added a test that checks the invitation email address.